/**:
  ros__parameters:
    antiwindup_cte : 1.0
    alpha: 0.3
    position_following:
      position_Kp: 
        x: 1.0
        y: 1.0
        z: 1.0
      position_Ki:
        x: 0.0000
        y: 0.0000
        z: 0.0000
      position_Kd: 
        x: 0.0
        y: 0.0
        z: 0.0
    yaw_speed_controller:
      Kp: 1.0
      Ki: 0.001
      Kd: 0.0